src/components/ModalDetails/ModalDetails.js

```diff
@@ -29,6 +29,7 @@ const ModalDetails = ({
   setMainDropdownChange,
   mainDropdownChange,
   setDisableExitBtnStatus,
+  disableColoums
```

Comment:
Although it does not negatively affect the change, "Coloums" in the field name should be "Columns" and I would argue for naming it "disabledColumns".  Minimally, correct the spelling.